### DISCUSSION TITLE
Access Issue Due to Lost Phone and 2FA

### DISCUSSION THREAD

**Discussion by baharbaba** [community/community#138104] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I’m unable to access my account due to a lost phone and the Microsoft Authenticator app. I’ve previously submitted ID verification for GitHub Student status and can resubmit if necessary. Additionally, I can provide proof of ownership for the website listed on my profile. Please advise on the steps to regain access.

Also, i have a discord account that connected to my github, my vscode still connected to my github, and i can push commit with git command

**Comment by VadymMakohon** [comment]
In your case, since you’ve lost access to your phone and Microsoft Authenticator, but you still have some access through tools like Visual Studio Code, Git, and Discord, there are a few steps you can follow to regain full access to your GitHub account.

### Steps to Recover Access:

1. **Submit a GitHub Support Request**:
   - Go to the [GitHub Support page](https://support.github.com/contact).
   - Select **Account Recovery** and provide all the relevant details: 
     - Your GitHub username.
     - The fact that your phone and 2FA method are lost.
     - Mention that you've previously submitted ID verification for GitHub Student status, and you're willing to resubmit it.
     - Provide details about your connected Discord account and your ability to still push commits via VS Code and Git, which can help verify ownership.

2. **Provide Proof of Ownership**:
   - Mention that your website is linked to your GitHub profile, and you can verify ownership of it.
   - Provide any additional identifying details about your account activity, repositories, or collaborators.

3. **Use Your Still-Connected Devices**:
   - **VS Code and Git**: Since you can push commits, you might still have an active authentication token or SSH key linked to your GitHub account. This won’t bypass the 2FA issue, but it’s worth mentioning to support as additional proof of account ownership.
   
4. **Prepare Your Identification**:
   - If GitHub requests further verification, be ready to resubmit any ID or proof related to your GitHub Student status.

5. **Disable 2FA via GitHub Support**:
   - GitHub Support will likely guide you through a process where they verify your identity, potentially disable 2FA on your account, or allow you to reset it with a new phone or authenticator app.

### Important:
- **Do not log out of VS Code** or any other session where you’re still authenticated. These could be useful for verifying your identity.
- Be patient, as account recovery can take some time depending on how much verification is needed.

By providing all this information to GitHub Support, you'll have a better chance of regaining access.

**Comment by baharbaba** [comment]
i opened a ticket its been 9 days but no response came i also provided proofs 5 days ago i need to use my github account i dont want to wait like 10000 days just for an account

**Comment by maryna-gol** [comment]
It’s frustrating to wait for that long without a response, especially when you're locked out of your account. Here are a few things you can try to help expedite the process:

1. Check Your Spam/Junk Folder:
Make sure GitHub’s responses haven’t been filtered into your spam or junk mail folder.
2. Follow Up on Your Ticket:
Respond to the support ticket politely asking for an update. You can reference the proofs you submitted and emphasize the urgency of accessing your account.
3. Contact GitHub via Social Media:
Sometimes, reaching out to GitHub Support via their [Twitter account (@githubsupport)](https://twitter.com/githubsupport) or other social media channels can help bring attention to your issue.
4. Ensure All Details Are Clear:
If you haven’t already, make sure your original ticket includes all relevant information:
Your GitHub username.
Description of your issue (lost 2FA, stolen phone, recovery attempts).
Proof of ownership (e.g., ID verification, access to services connected to GitHub).
5. Consider GitHub Premium Support:
If your account is tied to business-critical work, you could explore GitHub Premium Support, which offers faster responses. However, this is usually a paid option.

**Reply by baharbaba** [reply]
There is no github twitter account.

**Comment by baharbaba** [comment]
I gave up hope on GitHub, they lost me.

**Comment by baharbaba** [comment]
i found a way to clone my private repos:

https://drive.google.com/file/d/1N13qKE-CXqWOdsobvdAxY3evjiQmjjy_/view?usp=drive_link

**Comment by Harrylawal74** [comment]
try and request a one time password to your email and begin the recovery process. I think it takes a few verification steps but should work. Let me know though.

**Reply by baharbaba** [reply]
Already tried all buttons grayed.

**Comment by baharbaba** [comment]
Still no answer do i need to do something else to get respond??

**Comment by baharbaba** [comment]
I have got an response from github that says “ We have not forgotten about you, your ticket is still in queue”. I hope my problem get solved soon.

**Comment by baharbaba** [comment]
Its been 1 month and still waiting to get my portfolio back...
@github-staff

**Comment by baharbaba** [comment]
Do we have to be Karen to get an answer?
